docker/build.sh

```diff
@@ -17,6 +17,7 @@ echo "Git SHA: $GIT_SHA"
 
 docker build -f "$SCRIPT_DIR/Dockerfile" \
     --progress=plain \
+    -t "registry.pthome.de/splitpro:latest" \
```

Comment:
What is this ?

Reply:
Good finding :-) I pushed it to my local registry for testing and missed to exclude the changed build script in the PR